src/Carbon/Lang/fr.php

```diff
@@ -90,7 +90,7 @@
     'weekdays_min' => ['di', 'lu', 'ma', 'me', 'je', 've', 'sa'],
     'ordinal' => function ($number, $period) {
         switch ($period) {
-            // In french, only the first has be ordinal, other number remains cardinal
+            // In French, only the first has been ordinal, other number remains cardinal
```

Comment:
This is what I meant actually:
```suggestion
            // In French, only the first has to be ordinal, other number remains cardinal
```